src/V3Sched.cpp

```diff
@@ -254,8 +286,11 @@ void orderSequentially(AstCFunc* funcp, const LogicByScope& lbs) {
                             bodyp
                                 = new AstWhile{flp, new AstConst{flp, AstConst::BitTrue{}}, bodyp};
                         }
+                        subFuncp->addStmtsp(bodyp);
+                        splitCheck(subFuncp);
+                    } else {
+                        subFuncp->addStmtsp(bodyp);
                     }
-                    subFuncp->addStmtsp(bodyp);
```

Comment:
I think just adding a `splitCheck` after this line would be fine, it checks if the function has any statements. Less control flow is a bit more readable.

Reply:
Done.